File: VB/DiagramContainersVB/Models/Item.vb
Imports DevExpress.Web.ASPxDiagram

Public Class Item
    Public Property ID As Integer
    Public Property ContainerID As Integer?
    Public Property Type As DiagramShapeType
    Public Property Text As String
    Public Property X As Double
    Public Property Y As Double
    Public Property Width As Double
    Public Property Height As Double

    Public Sub New(ByVal id As Integer, ByVal containerId As Integer?, ByVal type As DiagramShapeType, ByVal text As String, ByVal x As Double, ByVal y As Double, ByVal width As Double, ByVal height As Double)
        Me.ID = id
        Me.ContainerID = containerId
        Me.Type = type
        Me.Text = text
        Me.X = x
        Me.Y = y
        Me.Width = width
        Me.Height = height
    End Sub

    Public Sub New()
    End Sub
End Class

File: CS/DiagramContainers/Models/Item.cs
using DevExpress.Web.ASPxDiagram;

namespace DiagramContainers.Models
{
    public class Item
    {
        public int ID { set; get; }
        public int? ContainerID { set; get; }
        public string Type { set; get; }
        public string Text { set; get; }
        public double X { set; get; }
        public double Y { set; get; }
        public double Width { set; get; }
        public double Height { set; get; }

        public Item(int id, int? containerId, string type, string text, double x, double y, double width, double height)
        {
            ID = id;
            ContainerID = containerId;
            Type = type;
            Text = text;
            X = x;
            Y = y;
            Width = width;
            Height = height;
        }

        public Item() { }
    }
}